lib/karafka/consumers/callbacks.rb

```diff
@@ -41,7 +41,7 @@ def self.included(controller_class)
         end
       end
 
-      # Executes the default controller flow, runs callbacks and if not halted will call process
+      # Executes the default consumer flow, runs callbacks and if not halted will call process
```

Comment:
Executes the default consumer flow, runs callbacks and if not halted will call process method of a proper backend. It is here because it interacts with the default Karafka call flow and needs to be overwritten to support callbacks